packages/embark-compiler/.gitignore

```diff
@@ -0,0 +1,4 @@
+.nyc_output
+/coverage
+/dist
+.DS_Store
```

Comment:
All of that can be at the root?

Reply:
I will explore this question (and related) in comments on the embarkjs PR #1294, there are a couple of things to consider.